tests/test_filters.py

```diff
@@ -37,26 +37,16 @@ def setUp(self):
 
     def test_singleton(self):
         self.filters1.defineFilters()
-        self.assertEqual(lsst.afw.image.Filter('abc').getFilterProperty().getLambdaEff(), 123)
-        self.assertEqual(lsst.afw.image.Filter('def').getFilterProperty().getLambdaEff(), 456)
-        self.filters1.defineFilters()  # this should not change anything
-        self.assertEqual(lsst.afw.image.Filter('abc').getFilterProperty().getLambdaEff(), 123)
-        self.assertEqual(lsst.afw.image.Filter('def').getFilterProperty().getLambdaEff(), 456)
```

Comment:
I think this test should stay for as long as `FilterDefinition` contains filter wavelengths (i.e., until [DM-27177](https://jira.lsstcorp.org/browse/DM-27177)). Same with the two tests below.